lib/resources/Source.js

```diff
@@ -4,8 +4,8 @@ var resource = require('../apiResources');
 
 var sources = function(config) {
   return resource.resourceActions('sources',
-    ['create'],
-    {'key': config['publicKey'], 'omiseVersion': config['omiseVersion']}
+    ['create', 'retrieve'],
+    {'key': config['secretKey'], 'omiseVersion': config['omiseVersion']}
```

Comment:
`create` use `publicKey` (even though, `secretKey` is acceptable here), but `retrieve` only permit using `secretKey`. So, the ideal solution would be using `publicKey` on `create` and `secretKey` on `retrieve`.